src/setup-docker/templates/fabric-docker/scripts/chaincode-functions-v2.sh

```diff
@@ -2,6 +2,8 @@
 # phrase "${CA_CERT_PARAMS[@]+"${CA_CERT_PARAMS[@]}"}" is needed in older bash versions ( <4 ) for array expansion.
 # see: https://stackoverflow.com/questions/7577052/bash-empty-array-expansion-with-set-u
 
+source chaincode-function-v1.4.sh
+
```

Comment:
We actually cannot do that. When Fablo generates scripts from templates it chooses whether it should be `chaincode-functions-v1.4.sh` or `chaincode-functions-v2.sh` file and renames it. If we use `v1.4`, then everything would work. If we use `v2`, then `v1.4` file would be missing. This is the case when duplication is perfectly fine. I suggest you to copy the function you created for `v1.4`. Also, could you fix indentation for the newly created function to meet our current style?

Reply:
also note we use camel case for function names

Reply:
Okay got it even I was bit surprised to see some code duplication, but thanks for the clarification